.utility/generate_rst_from_md.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+"""Script to generate a rst file automatically from an markdown file.
+
+Usage
+-----
+generate_rst_from_md.py [-h] markdown rst
+
+positional arguments:
+  markdown    Path of the source markdown file
+  rst         Path where generated rst should be stored
+
+optional arguments:
+  -h, --help  show this help message and exit
+"""
+import argparse
+import pypandoc
+
+def download_pandoc():
+    """Download pandoc if not already installed"""
+    try:
+        # Check whether it is already installed
+        pypandoc.get_pandoc_version()
+    except OSError:
+        # Pandoc not installed. Let's download it.
+        pypandoc.download_pandoc()
+
+        # Hack to delete the downloaded file from the folder,
+        # otherwise it could get accidently committed to the repo
+        # by other scripts in the repo.
+        import sys
+        import os
```

Comment:
@pri22296 yep, importing sys is fine, but since you are only using platform, so you could do 
`from sys import platform`
this prevents unnecessary imports from sys library